docs/usage/shoot_access.md

```diff
@@ -79,3 +54,30 @@ If you want to use the same OIDC configuration for all your shoots by default, t
 Shoots have to "opt-in" for such defaulting by using the `oidc=enable` label.
 
 For further information on `(Cluster)OpenIDConnectPreset`, refer to [ClusterOpenIDConnectPreset and OpenIDConnectPreset](openidconnect-presets.md).
+
+## Static Token kubeconfig 
+
+> **Note:** Static token kubeconfig is not available for Shoot clusters created with Kubernetes version >= 1.27. The [`shoots/adminkubeconfig` subresource](#shootsadminkubeconfig-subresource) should be used instead.
```

Comment:
Btw, it is not only about Shoot clusters created with Kubernetes version >= 1.27. It should also forbid updating from K8s 1.26 to 1.27 when enableStaticTokenKubeconfig=true. Anyways, this validation is about to be added as part of https://github.com/gardener/gardener/issues/7783.

```suggestion
> **Note:** Static token kubeconfig is not available for Shoot clusters with Kubernetes version >= 1.27. The [`shoots/adminkubeconfig` subresource](#shootsadminkubeconfig-subresource) should be used instead.
```
or 

```suggestion
> **Note:** Static token kubeconfig is not available for Shoot clusters using Kubernetes version >= 1.27. The [`shoots/adminkubeconfig` subresource](#shootsadminkubeconfig-subresource) should be used instead.
```